requirements/edx/github.in

```diff
@@ -88,7 +88,7 @@
 # Our libraries:
 -e git+https://github.com/edx/codejail.git@a320d43ce6b9c93b17636b2491f724d9e433be47#egg=codejail
 -e git+https://github.com/edx/acid-block.git@e46f9cda8a03e121a00c7e347084d142d22ebfb7#egg=acid-xblock
--e git+https://github.com/edx/edx-ora2.git@2.2.1#egg=ora2==2.2.1
+-e git+https://github.com/edx/edx-ora2.git@2.2.2#egg=ora2==2.2.2
```

Comment:
Why does base.txt have no `-e`, but github.in has `-e`?  Is that right?

Reply:
From `github.in`:

```python
#   * The "-e" prefix is primarily for the benefit of pip-compile, which does
#     not yet support non-editable GitHub URLs.  If a VERSION is correctly
#     specified as shown above, the "-e" prefix will be stripped from the
#     output file by the post-pip-compile.sh script. (Note that we previously
#     recommended omitting this prefix, and will again if the relevant
#     pip-tools issue is fixed.)
```

Reply:
Who reads the tops of files!? :)